test/functional/legacy/listlbr_utf8_spec.lua

```diff
@@ -1,209 +1,184 @@
-      call DoRecordScreen()
-      " expected: attr[2] is different because of colorcolumn
-      if attr[0] != attr2[0] || attr[1] != attr2[1] || attr[2] != attr2[2]
-         call append('$', "Screen attributes are different!")
-      else
-         call append('$', "Screen attributes are the same!")
-      endif
+    syn clear
+    syn match meaning    /;\s*\zs.*/
+    syn match hasword    /^\x\{8}/    contains=word
+    syn match word       /\<\x\{8}\>/ contains=beginword,endword contained
+    syn match beginword  /\<\x\x/     contained conceal
+    syn match endword    /\x\{6}\>/   contained
+    hi meaning   guibg=blue
+    hi beginword guibg=green
+    hi endword   guibg=red
     ]])
+    helpers.wait()
+    screen:expect("eeeeee>--->-;>some text",
+    nil,nil,nil, true )
+  end)
 
-    -- Assert buffer contents.
-    expect([[
-      
-      	abcdef hijklmn	pqrstuvwxyz 1060ABCDEFGHIJKLMNOP 
-      
-      Test 1: set linebreak + set list + fancy listchars
-      ▕———abcdef          
-      +hijklmn▕———        
-      +pqrstuvwxyz␣1060ABC
-      +DEFGHIJKLMNOPˑ¶    
-      
-      Test 2: set nolinebreak list
-      ▕———abcdef hijklmn▕—
-      +pqrstuvwxyz␣1060ABC
-      +DEFGHIJKLMNOPˑ¶    
-      ¶                   
-      	*mask = nil;
-      
-      Test 3: set linebreak nolist
-          *mask = nil;    
-      ~                   
-      ~                   
-      ~                   
-      #define ABCDE		1
-      #define ABCDEF		1
-      #define ABCDEFG		1
-      #define ABCDEFGH	1
-      #define MSG_MODE_FILE			1
-      #define MSG_MODE_CONSOLE		2
-      #define MSG_MODE_FILE_AND_CONSOLE	3
-      #define MSG_MODE_FILE_THEN_CONSOLE	4
-      
-      Test 4: set linebreak list listchars and concealing
-      #define ABCDE>-->---1                   
-      #define >CDEF>-->---1                   
-      #define >CDEFG>->---1                   
-      #define >CDEFGH>----1                   
-      #define >_FILE>--------->--->---1       
-      #define >_CONSOLE>---------->---2       
-      #define >_FILE_AND_CONSOLE>---------3   
-      bbeeeeee		;	some text
-      
-      Test 5: set linebreak list listchars and concealing part2
-      eeeeee>--->-;>some text                 
-      Test 6: Screenattributes for comment
-       /*		 and some more */
-      ScreenAttributes for test6:
-      Attribut 0 and 1 and 3 and 5 are different!
-      Test 8: set linebreak with visual block mode and v_b_A and selection=exclusive and multibyte char
-      long line: foobar foobar foobar foobar foobar foobar foobar foobar foobar foobar foobar foobar foobar foobar foobar foobar foobar foobar foobar foobar foobar foobar foobar foobar foobar foobar foobar foobar foobar foobar foobar foobar foobar foobar foobar foobar foobar foobar foobar foobar TARGETÃx' at end
+  -- TODO update with new neovim system to check attributes
+  -- or remove it if done somewhere else
+  -- it("screenattributes for comment", function()
```

Comment:
seems like a bad test compared to screen:expect . do you still want me to upgrade it ? or simply remove it ?